imports/ui/pages/professor/questions_public.jsx

```diff
@@ -153,8 +154,9 @@ class _QuestionsPublic extends Component {
 
 }
 
-export const QuestionsPublic = createContainer(() => {
+export const QuestionsPublic = createContainer(props => {
   const handle = Meteor.subscribe('questions.public')
```

Comment:
update the subscription to take a courseId argument.

I assume that questions from student also needs to be updated (component and subscription)